ord_schema/visualization/generate_text.py

```diff
@@ -101,5 +95,12 @@ def main(argv):
 
 
 if __name__ == '__main__':
+    FLAGS = flags.FLAGS
```

Comment:
Not necessarily required for this PR, but I think it would be good to convert this module to a library rather than a script. These flags can be converted to function args, for instance, and then removed.